api/scripts/server.ts

```diff
@@ -0,0 +1,19 @@
+import fastify from 'fastify';
```

Comment:
This file can be called `serve.ts` IMO

Reply:
Do you not agree @reishaleem ?

Reply:
I don't. You can change it in the future if you want